lib/perl/Genome/InstrumentData/Command/AlignAndMerge.pm

```diff
@@ -162,7 +139,7 @@ sub _process_alignments {
     my $self = shift;
     my $mode = shift;
 
-    my $class = $self->merged_result_class;
+    my $class = $self->class->merged_result_class($self->name);
```

Comment:
The introduction of the intermediate `class` in this chain makes the variable name `$class` seem more confusing.  Perhaps it could be `$merged_class` (and later `$per_lane_class`)?